### ISSUE TITLE
Adopt or Align with open API Stand

### ISSUE THREAD

**Issue by samperd** [wet-boew/wet-boew-api-standards#41]
Since we last visited this issue a few more environmental drivers have popped up.

I propose we either re-examine this standard and align more closely with the Open API Standard (https://www.openapis.org/). Or alternatively we just adopt the Open API Standard.

**Comment by gnewton** [comment]
I would vote for just adopting  Open API Standard. With respect to #42, I would prefer that OAS was chosen, as opposed to Swagger (are they different?). From https://www.openapis.org/faq#OAIFAQ-Swagger it is not clear (to me) how Swagger and OAS will move forward, and if they might diverge. If they do not diverge, then adopting OAS is not a problem. If they do diverge and we have chosen Swagger but most of industry chooses OAS, then it is a problem...

**Comment by LaurentGoderre** [comment]
I agree with @gnewton I would rather we adhere to a Standard than an implementation
